### DISCUSSION TITLE
Parts Edit - Requires Link to External URL

### DISCUSSION THREAD

**Discussion by imager2019** [inventree/InvenTree#8081] [General]
Running 0.16.1 and noticed today while performing an edit on a part, that the page is forcing an URL to be entered on the Link field (Link to external URL).

Took a look thought the site settings, but could not locate where this is being forced to be entered. 

Is there a setting somewhere that I have missed that I can turn off so that a URL is not required to be entered??

![image](https://github.com/user-attachments/assets/b3962ce6-5b93-4ac0-8718-201577b4727b)

**Comment by SchrodingersGat** [comment]
### Related Issues / Discussions

- https://github.com/inventree/InvenTree/issues/8039
- https://github.com/inventree/InvenTree/discussions/8041

It is interesting that this has now shown up three times. I have not been able to reproduce this issue locally, or on the demo server.

@imager2019 can you please provide the full information on the request / response when you hit the "submit" button?

This is available in the developer tab in your browser. I am interested in:

- The full data being sent to the server
- The full response from the server

**Reply by imager2019** [reply]
@SchrodingersGat .. I can assist, but not familiar with the Developer mode. I am in the console, but unsure how to get the request/response information when the submit button is pressed. Can you give me some quick details on how to capture this??

Below is what my Developer Console looks like..

![image](https://github.com/user-attachments/assets/5c66f6ff-9dc0-423c-8d5b-2553fb6186f8)

**Reply by SchrodingersGat** [reply]
Ok, so:

1. Locate the "network" tab, and filter request type by "Fetch/XHR":

![image](https://github.com/user-attachments/assets/49612aa9-d2a2-4a28-918d-68b35d49a942)

2. Hit "SUBMIT" and look for the transaction that has status code "400"

3. Click on that transaction to bring up detail view

4. Screenshot the "payload" tab

5. Screenshot the "Response" tab

**Reply by imager2019** [reply]
OKay... Here is the capture on Chrome on Windows:
![image](https://github.com/user-attachments/assets/f48b83cc-4dd2-4983-bda0-8b3129909cfd)
![image](https://github.com/user-attachments/assets/c65efee3-a9e9-4c0b-8b41-53e795a98fed)

Also did the same, but with Firefox on Ubuntu
![image](https://github.com/user-attachments/assets/1a3ab4f9-429c-4520-9db9-f34ec74d3712)
![image](https://github.com/user-attachments/assets/18cce55f-9e16-4a8e-ab60-14e264e6fb7a)

**Reply by SchrodingersGat** [reply]
Thanks!

I have rolled back my installation to `0.16.1`, and I still cannot reproduce.

Can you try  pulling down the server, running the following commands on your installation, and then restarting the server and see if the problem persists:

- `invoke update`
- `invoke migrate`
- `invoke install`

**Reply by imager2019** [reply]
I need to run the commands as follows on my install.

```
- inventree run cli update
- inventree run cli migrate
- inventree run cli install
```

Same results with the Enter a valid URL request.....
